packages/internet-header/src/assets/i18n/translations.ts

```diff
@@ -1,4 +1,9 @@
 export const translations = {
+  'Active navigation point': {
+    de: 'Aktive Navigationspunkt',
+    fr: 'Point de navigation actif',
+    it: 'Punto di navigazione attivo',
```

Comment:
```suggestion
    it: 'Voce di menu attiva',
```

Reply:
I changed this to "Elemento di navigazione attivo" to be inline with the other languages.